client/app/application.coffee

```diff
@@ -87,9 +78,21 @@ class App extends Application
 
 
     # Handler for register route, display onboarding's current step
-    handleRegisterRoute: =>
+    initializeOnboarding: =>
         # Load onboarding stylesheet
         AppStyles = require './styles/onboarding.styl'
+        # onboarding steps config
+        steps = require './config/steps/all'
+
+        user = {
+            username: ENV.username,
+            hasValidInfos: ENV.hasValidInfos
+        }
+
+        @onboarding = new Onboarding(user, steps, ENV.currentStep)
```

Comment:
You're right, I didn't go far enought in the split of all the onboarding code here, that still called the onboarding initialize functions each time the steps changed. Thanks! 👍